dropwizard-jersey/src/main/java/io/dropwizard/jersey/params/IntParam.java

```diff
@@ -5,7 +5,10 @@
 /**
  * A parameter encapsulating integer values. All non-decimal values will return a
  * {@code 400 Bad Request} response.
+ * 
+ * @deprecated As of release 1.3.9, will be removed in 2.0.0. Please use {@link java.util.OptionalInt} instead.
```

Comment:
Will `OptionalInt` be a true replacement in v1.3.9? I thought it could only be replaced with the new functionality brought in Jersey 2.28 + hibernate validator 6.0

Reply:
I'll remove that mention